src/status_im/contexts/chat/messages/content/audio/component_spec.cljs

```diff
@@ -1,28 +1,26 @@
 (ns status-im.contexts.chat.messages.content.audio.component-spec
   (:require
-    [re-frame.core :as re-frame]
     [react-native.audio-toolkit :as audio]
     [status-im.contexts.chat.messages.content.audio.view :as audio-message]
     [test-helpers.component :as h]))
 
+;; We can't rely on `with-redefs` with async code.
+(set! audio/destroy-player #())
```

Comment:
FYI: Using `set!` is only safe because Jest isolates tests between files, so the `audio/destroy-player` var is modified only within the scope of the tests in this file.